resolwe_bio/docker_images/legacy/packages-debian.txt

```diff
@@ -0,0 +1,56 @@
+bedtools
+samtools
+picard-tools
+bowtie
+bowtie2
+bwa
+tophat
+abyss
+# required by abyss scripts
```

Comment:
How many of the packages below are required by `abyss`?

Please, use new lines to structure the contents of the file. Also add comments on why something is needed where is appropriate.